docs/working-agreement.md

```diff
@@ -0,0 +1,79 @@
+# Sections of a Working Agreement
+
+A working agreement is a document, or a set of documents that describe how we work together as a team and what our
+expectations and principles are.
+
+## General  
+
+- We work as one team towards a common goal and clear scope
+- We make sure everyone's voice is heard, listened to
+- We show all team members equal respect
+- We work as a team to have common expectations for technical delivery
+- We make sure to spread our expertise and skills in the team, so no single person is relied on for one skill
+- All times below are listed in PST
+
+## Communication
+
+- We communicate all information relevant to the team through the Project Slack channel
+- We add all technical spikes, trade studies, and other technical documentation to the project repository through async design reviews in PRs.
+
+## Work-life Balance
+
+- We work in a volunteering org and respect this, especially when setting up recurring meetings.
```

Comment:
Team members contribute minimum of 4 hours per week

Reply:
I added the 4 hours per week. Thanks!